### PR TITLE
COM-287 Patient Reg: Relationship type Change Request implemented

### PR DESCRIPTION
**Pull request by lauravignoli**

### CODE REVIEW

File: openmrs/i18n/clinical/locale_fr.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
   "RETROSPECTIVE_POPUP_ON_BEHALF_LABEL": "Capturer information en tant que",
   "RETROSPECTIVE_POPUP_DATA_FOR_LABEL": "Capturer information pour la date du",
   "OKAY_LABEL": "OK",
-  "RELATIONSHIP_SPOUSE": "Époux",
+  "RELATIONSHIP_PARTNER": "partenaire",
```
Comment (lauravignoli) [inline openmrs/i18n/clinical/locale_fr.json]: @IipperMangesh Please use capital P

File: openmrs/i18n/adt/locale_fr.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
   "RELATIONSHIP_CAREGIVER": "Soignant",
   "RELATIONSHIP_AUNT_UNCLE": "Tante/Oncle",
   "RELATIONSHIP_OTHER": "Autre",
-  "RELATIONSHIP_SPOUSE": "Époux",
+  "RELATIONSHIP_PARTNER": "partenaire",
```
Comment (lauravignoli) [inline openmrs/i18n/adt/locale_fr.json]: @IipperMangesh Please use capital P

File: openmrs/i18n/registration/locale_fr.json [hunk 3]
```diff
@@ -102,7 +102,7 @@
   "RELATIONSHIP_CAREGIVER": "Soignant",
   "RELATIONSHIP_AUNT_UNCLE": "Tante/Oncle",
   "RELATIONSHIP_OTHER": "Autre",
-  "RELATIONSHIP_SPOUSE": "Époux",
+  "RELATIONSHIP_PARTNER": "partenaire",
```
Comment (lauravignoli) [inline openmrs/i18n/registration/locale_fr.json]: @IipperMangesh Please use capital P

File: openmrs/i18n/adt/locale_fr.json (hunk 2 above)
Comment (IipperMangesh) [reply openmrs/i18n/adt/locale_fr.json]: @lauravignoli  Update locale_fr.json translation.

File: openmrs/i18n/registration/locale_fr.json (hunk 3 above)
Comment (IipperMangesh) [reply openmrs/i18n/registration/locale_fr.json]: @lauravignoli  Update locale_fr.json translation.

File: openmrs/i18n/clinical/locale_fr.json (hunk 1 above)
Comment (IipperMangesh) [reply openmrs/i18n/clinical/locale_fr.json]: @lauravignoli  Update locale_fr.json translation.

File: openmrs/i18n/adt/locale_en.json [hunk 4]
```diff
@@ -60,7 +60,7 @@
   "RELATIONSHIP_CAREGIVER": "Caregiver",
```
Comment (lauravignoli) [inline openmrs/i18n/adt/locale_en.json]: @IipperMangesh Please remove RELATIONSHIP_DOCTOR, RELATIONSHIP_PATIENT, RELATIONSHIP_SUPERVISOR, RELATIONSHIP_SUPERVISEE, RELATIONSHIP_CAREGIVER from all translation files as they are not being used anymore.

**Pull request merged**